followthemoney/proxy.py

```diff
@@ -329,6 +329,28 @@ def countries(self) -> List[str]:
         """Get the set of all country-type values set of the entity."""
         return self.get_type_values(registry.country)
 
+    @property
+    def temporal_start(self) -> Optional[str]:
```

Comment:
I think I might actually change this (and the respective methods in the TS package) to return a tuple and include the specific property that corresponds to the value returned.

When displaying entities in a timeline, we probably want to display details when users interact with entities, including the label of the start and end property.

Reply:
I can sort of see how this solves the problem, but my imagination (of the UI code) doesn't yet expand to a place where this problem occurs and isn't fixable otherwise. KISS?

Reply:
Fair point. The use case for this is displaying both the value and property name of the temporal start for a property:

<img width="453" alt="Screen Shot 2022-11-17 at 15 44 14" src="https://user-images.githubusercontent.com/1512805/202476966-719c69b0-5caa-4ee3-aa75-e26ba476ec8e.png">

It is fixable otherwise, but I think that would mean basically duplicating most of the logic in the Aleph UI, which I’d like to avoid. Any ideas?

Reply:
Sold. I vote we merge this whole thing!